docs/apm/traces/working-with-tracing-data/service-map.md

```diff
@@ -1,64 +1,96 @@
 
-![service map manual detection settings.png](/img/traces/service-map-manual-detection-settings.png)
+<img src={useBaseUrl('img/traces/settings-general.png')} alt="service map gear icon for settings" width="450"/>
 
-## Service Map Dashboard Panels
 
-You can add Service Map panels to explore your application environment directly through your Dashboard. Duplicate or add multiple panels with different filtering or queries to refine views and support your organization. 
+### Services Dashboard Panels
 
-The Service Map panel displays a service map for filtered applications and services, with the ability to explore the environment. To add a Service Map panel to your Dashboard:
+The **Services Map** and **Services List** panels, available out of the box, enable you to explore your application environment and review all traces from your Dashboard. You can duplicate or add multiple panels with different filtering or queries to refine views and support your organization.
 
-1. Open a Dashboard or create a new Dashboard.
-1. Click **Add Panel** and select **Service Map**. Or select the **Service Map** option on a new Dashboard.<br/> ![add-servicemap.png](/img/traces/add-servicemap.png)  
-1. A panel configuration page opens.  <br/> ![create-servicemap.png](/img/traces/create-servicemap.png)
-1. Select from the dropdown menus to filter the Service Map by the following:<br/>  ![filter-servicemap.png](/img/traces/filter-servicemap.png)
-   * **Application** if your tracing data has the `application=[app-name]` tag
-   * **Service**
-   * To pass the variables from dashboard filters, set `application={{application}}` and/or `service={{service}}`
-1. Do not set the time. Service Map always shows last 72h of data.
-1. The **Chart Type** is set to Graph.
-1. Click the **General** tab to edit the Panel Details. Enter a name for the panel, set a **Title Font Size**, and add a short **Description**. <br/>![tracelist-details.png](/img/traces/tracelist-details.png)
-1. Click **Add to Dashboard**. The panel loads in your Dashboard to review your applications and services according to filtering.<br/>  ![panel-service.png](/img/traces/panel-service.png)
+To add a Services panel to your Dashboard:
+
+1. Open an existing Dashboard or create a new Dashboard (**+New** > **Dashboard New**).
+1. On an existing Dashboard, click **Add Panel** > **Services**, or, on a new Dashboard, click **Services**.<br/> ![add-servicemap.png](/img/traces/add-servicemap.png)  
+1. When the panel configuration page opens, go to the **Visual Settings** > **Chart Type**, dropdown and select **Table** (Services List) or **Graph** (Services Map). <br/> ![create-servicemap.png](/img/traces/create-servicemap.png)
+1. Select from the dropdown menus to filter by the following:<br/>  ![filter-servicemap.png](/img/traces/filter-servicemap.png)
+   * **Application** (Optional). You can use this if your tracing data has the `application=[app-name]` tag
```

Comment:
On my local yarn instance this line breaks into new on in the middle of the app-name tag.